src/states/SelectAttackState.py

```diff
@@ -48,14 +120,25 @@ def update(self, dt, events):
 
     def render(self, screen):
         # Turn
-        screen.blit(pygame.font.Font(None, 36).render(f"SelectAttackState - Turn {self.turn}", True, (0, 0, 0)), (10, 10))   
+        screen.blit(pygame.font.Font(None, 36).render(f"SelectAttackState - Turn {self.turn} - {self.effectOwner}", True, (0, 0, 0)), (10, 10))   
 
         # Render cards on player's hand
         for order, card in enumerate(self.player.cardsOnHand):
             card.render(screen, order)
 
         # Render field
-        for fieldTile in self.field:
-            fieldTile.render(screen, len(self.field))
+        for fieldTile in self.field:               
+            # Render the range of the attack
 
-        
\ No newline at end of file
+            if fieldTile.index in set(self.avilableAttackTile):
+                fieldTile.color = (255,0,0)
+            else:
+                fieldTile.color = (0,0,0)
+            if self.selectAttackTile>=0:
+                if fieldTile.index == self.avilableAttackTile[self.selectAttackTile]:
+                    fieldTile.color = (255,0,255)
+                    fieldTile.solid = 0
```

Comment:
will it be better if we define the color as constant variable for future change?